canonical-web-and-design/certification.ubuntu.com
# Support /components/hardware/{canonical_id} endpoints

QA
--

`./run`, go to e.g. http://localhost:8034/components/hardware/201605-22234/, see it redirect and filter by `canonical_id`.
